readiness-tool/popup.js

```diff
@@ -134,25 +141,21 @@ function filterApps(htmlString, listAllApps) {
             'is not declared as an ads or analytics vendor in apps.json');
           return;
         }
-        addToDict(x.split(appsRegexDelimeter)[0], htmlString, filteredApps, appName,
-          appConfig.cats[0]);
+        addToDict(x.split(appsRegexDelimeter)[0], htmlString, filteredApps,
+                  appName, appConfig.cats[0]);
       });
     }
   });
   return filteredApps;
 }
 
 /**
- * An object contained by 2 more objects that each hold two string arrays
- * @typedef {{[String][String]}{[String][String]}} filteringObject
- */
-
-/**
  * Pushes app names to the supported or not supported list of the object
- 'found this'
+ * 'filtedApss'
  * @param {string} regexString - String representation of regular expression
  * @param {!string} htmlString - String containing all HTML on the page
- * @param {filteringObject} filteredApps - Object separating the 3P services by support
+ * @param {FilteredAppsDef} filteredApps - Object separating the 3P services by 
```

Comment:
👍 

Also update the type declarations in other functions that return or take this object as a param.